ai-platform-unified/notebooks/official/featurestore_movie_prediction.ipynb

```diff
@@ -0,0 +1,1372 @@
+{
```

Comment:
<p>"The task is to prepare a training dataset..." (remove "for")</p><p> </p><p>"...from Feature Store according to the..." (I think we're referring to Feature Store the service here)</p><p> </p><p>"...feature values from the featurestore..." (add "the")</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/283/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>